docs/conf.py

```diff
@@ -25,42 +24,57 @@
 extensions = [
     "sphinx.ext.napoleon",
     "sphinx.ext.autodoc",
-    "sphinx.ext.autosummary",
     "sphinx.ext.mathjax",
     "sphinx.ext.viewcode",
     "sphinx.ext.extlinks",
     "sphinx.ext.intersphinx",
+    "sphinxcontrib.autodoc_pydantic",
     "jupyter_sphinx",
+    "qiskit_sphinx_theme",
 ]
-templates_path = ["_templates"]
-html_static_path = ["_static"]
-html_css_files = []
 
-autosummary_generate = True
-autosummary_generate_overwrite = False
-autoclass_content = "both"
+# --------------------
+# Theme
+# --------------------
 
-numfig = True
+html_theme = "qiskit-ecosystem"
+pygments_style = "emacs"
```

Comment:
This makes it so the left side bar doesn't include the word "documentation" in the site title:

```suggestion
html_theme = "qiskit-ecosystem"
pygments_style = "emacs"
html_title = f"{project} {release}"
```

Reply:
Sorry, your review crossed the merge process. Could you please file a PR with your change?